src/darwin/Framework/CHIP/templates/templates.json

```diff
@@ -11,7 +11,8 @@
         "darwin/Framework/CHIP/templates/helper.js"
     ],
     "resources": {
-        "availability-data": "availability.yaml"
+        "availability-data": "availability.yaml",
+        "config-data": "../../../../app/common/templates/config-data.yaml"
```

Comment:
Could just add a config-data YAML to this directory, but this is fine too.

Reply:
Let's do updates as followups, given that I was told having this in is reasonably important to unbreak some CI.